nixos/modules/services/networking/gdomap.nix

```diff
@@ -11,22 +11,10 @@ in
   #
   options = {
     services.gdomap = {
-      enable = mkOption {
-        default = false;
-	description = "
-	  Whether to enable gdomap, the GNUstep distributed objects daemon.
-
-	  Note that gdomap runs as root.
-        ";
-      };
-
-      pidfile = mkOption {
-        type = types.path;
-        default = "/tmp/gdomap.pid";
-	description = "Location of the pid file for gdomap daemon";
-      };
+      enable = mkEnableOption "Whether to enable gdomap, the GNUstep distributed objects daemon";
```

Comment:
Note that `mkEnableOption` expects the parameter to be the name of the service; I believe this will expand to a description like "Whether to enable Whether to enable gdomap".